mesh/storage.c

```diff
@@ -496,6 +446,8 @@ bool storage_load_nodes(const char *dir_name)
 
 		if (parse_config(name_buf, filename, node_id))
 			continue;
+		else
+			l_info("Cannot parse config file (incorrect JSON format)");
```

Comment:
Would it be possible to get parse error mesage from json-c?

Reply:
Yes it is possible, done.